### ISSUE TITLE
linking research notes

### ISSUE THREAD

**Issue by mmclaugh0000** [jywarren/plots#149]
hello, it would be great if we could link research notes. thanks

**Comment by jywarren** [comment]
Cool idea, like list some related notes when you post?
On Apr 19, 2012 7:01 PM, "mmclaugh0000" <
reply@reply.github.com>
wrote:

**Comment by mmclaugh0000** [comment]
yep, specifically for the hydrogen sulfide experiments...i posted the
experiment set up separate from results but they kinda need to be seen
together to make sense.

On Thu, Apr 19, 2012 at 7:03 PM, Jeffrey Warren <
reply@reply.github.com

> wrote:
> 
> Cool idea, like list some related notes when you post?
> On Apr 19, 2012 7:01 PM, "mmclaugh0000" <
> reply@reply.github.com>
> wrote:
> 
> > hello, it would be great if we could link research notes. thanks
> > 
> > ---
> > 
> > Reply to this email directly or view it on GitHub:
> > https://github.com/jywarren/plots/issues/149
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/jywarren/plots/issues/149#issuecomment-5234590

## 

Megan McLaughlin
mmclaugh@risd.edu
c. 401.527.5528
